chromium_src/chrome/browser/ui/ui_features.cc

```diff
@@ -12,6 +12,9 @@ namespace features {
 
 OVERRIDE_FEATURE_DEFAULT_STATES({{
     {kChromeLabs, base::FEATURE_DISABLED_BY_DEFAULT},
+#if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX)
+    {kFewerUpdateConfirmations, base::FEATURE_DISABLED_BY_DEFAULT},
```

Comment:
Do we want to add this to the disabled features unit test?

Reply:
Yep, good catch.

Reply:
Added